itest/itest_lndharness_test.go

```diff
@@ -366,3 +370,28 @@ func mustGenCertPair(t *testing.T, certFile, keyFile string) {
 
 	require.NoError(t, os.WriteFile(keyFile, keyBuf.Bytes(), 0600))
 }
+
+func mustGenMacaroon(t *testing.T, macPath string) {
```

Comment:
I think you can squash the itest commits to a single one.

Reply:
Done. :heavy_check_mark: